tests/Oryx.Integration.Tests/LocalDockerTests/PythonEndToEndTests.cs

```diff
@@ -583,7 +583,7 @@ public async Task CanBuildAndRun_ShapelyFlaskApp_UsingVirtualEnv(string pythonVe
                 _output,
                 volume,
                 "oryx",
-                new[] { "build", appDir },
+                new[] { "build", appDir, "-l", "python", "--language-version", pythonVersion },
```

Comment:
I think this test should fail as this test seems to be passing `virtualEnvName` to startup script path for runtime. Probably it was failing even before your changes?

Reply:
It's fine to pass the virtual env name to the startup script. It won't actually use it though, as it will take it from the manifest file.
It was passing before, becase using target folder is more forgiving to version differences. Since virtual envs actually contain the python binaries, it created problems when build and runtime are different